services/sam-audio/server.py

```
@@ -0,0 +1,292 @@
+    if _sam_bundle is not None:
+        return _sam_bundle
+    device = _resolve_device()
+    try:
+        from sam_audio import SAMAudio, SAMAudioProcessor  # type: ignore
+
+        model = SAMAudio.from_pretrained(MODEL_ID)
+        processor = SAMAudioProcessor.from_pretrained(MODEL_ID)
+        if device == "cuda":
+            model = model.eval().cuda()
+        else:
+            model = model.eval()
```

Ruff (0.16.1)

[warning] Use ternary operator `model = model.eval().cuda() if device == "cuda" else model.eval()` instead of `if`-`else`-block

Replace `if`-`else`-block with `model = model.eval().cuda() if device == "cuda" else model.eval()`

(SIM108)